tests/jballerina-unit-test/src/test/resources/test-src/annotations/annot_attachments_negative.bal

```diff
@@ -975,3 +975,24 @@ type F4 record {|
 type F5 record {|
     int x;
 |};
+
+function testInvalidAnnotationAttachmentOnField() {
+    [@UndefinedAnnotation int, int] [first, second] = [1, 2];
+    [@UndefinedAnnotation int, int, int] [a, b, c] = [1, 2, 3];
+    [[@UndefinedAnnotation int, int], int] [[a1, b1], c1] = [[1, 2], 3];
+    record {|@annot string fname; string lname;|} {fname, lname} = getPerson();
+}
+
+type Person record {|
+    string fname;
+    string lname;
+|};
+
+function getPerson() returns Person {
+    Person person = {fname: "Anne", lname: "Frank"};
+    return person;
+}
+
+[@UndefinedAnnotation int, int] [w, e] = [1, 2];
+
+error<record {| @UndefinedAnnotation int i; |}> err = error("hi", i = 33);
```

Comment:
While it's just a test, better to use appropriate values. E.g., err! instead of hi as the error message.